WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -379,17 +379,27 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
     /// Presents the Support Interface from a given ViewController, with a specified SourceTag.
     ///
     func presentSupport(from sourceViewController: UIViewController, sourceTag: WordPressSupportSourceTag) {
```

Comment:
nit: do we still need this outdated delegate function?

Reply:
Oh, this one is from [the Authenticator protocol](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/ServiceLocator/Authentication.swift#L11). And, yes, we need this. 

I will change this in future when dealing with screens from WCiOS. (Not from the WordPressAuthenticator library)